addons/crm/models/res_partner.py

```diff
@@ -19,7 +19,7 @@ class Partner(models.Model):
     def default_get(self, fields):
         rec = super(Partner, self).default_get(fields)
         active_model = self.env.context.get('active_model')
-        if active_model == 'crm.lead':
+        if active_model == 'crm.lead' and (not self.env.context.get('active_ids') or len(self.env.context.get('active_ids')) <= 1):
```

Comment:
I think this is slightly shorter and gives the same result.
```suggestion
        if active_model == 'crm.lead' and len(self.env.context.get('active_ids', [])) <= 1:
```